fbw-a380x/src/base/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/aircraft.cfg

```diff
@@ -6,15 +6,15 @@ major = 1
 minor = 0
 
 [GENERAL]
-atc_type = "TT:ATCCOM.ATC_NAME AIRBUS.0.text"
-atc_model = "TT:ATCCOM.AC_MODEL A388.0.text"
+atc_type = "Airbus"
+atc_model = "FBWA380X"
 Category = "airplane"
 performance = "Todo"
 editable = 1
 wip_indicator = 2
 icao_type_designator = "A388"
 icao_manufacturer = "AIRBUS"
-icao_model = "A-380-800"
+icao_model = "A380X"
```

Comment:
This was correct before https://www.icao.int/publications/doc8643/pages/search.aspx